### PR TITLE
Add SortTabs component on the FE for sorting Search Queries

### LINKED ISSUE #605: Feat: Implement Sorting by Price/Quality/Duration using SortTabs Component
Implement the ability to sort by price quality or duration by using the `SortTabs` component

### PR DESCRIPTION
**Pull request by aalises**
Closes #605

### CODE REVIEW

File: apps/core/scenes/search/SearchContext.js [hunk 1]
```diff
@@ -26,12 +26,13 @@ export type Location = {|
...
 
 export type LocationSearchType = 'travelTo' | 'travelFrom';
-
+export type SortTypes = 'QUALITY' | 'PRICE' | 'DURATION';
 type State = {|
   tripType: TripTypes,
   travelFrom: Array<Location>,
   travelTo: Array<Location>,
   dateFrom: Date,
+  sort: SortTypes,
```
Comment (tbergq) [inline apps/core/scenes/search/SearchContext.js]: could this be named `sortBy` instead?

File: apps/core/scenes/search/SortTabsWrapper.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+import {
+  withSearchContext,
+  type SearchContextState,
+  type SortTypes,
+} from './SearchContext';
+
+type Props = {
+  sort: SortTypes,
+  setSortBy: SortTypes => void,
```
Comment (tbergq) [inline apps/core/scenes/search/SortTabsWrapper.js]: Please use extact type, and read-only props

File: apps/core/scenes/search/SearchContext.js (hunk 1 above)
Comment (aalises) [reply apps/core/scenes/search/SearchContext.js]: @tbergq sure!! Should we also change it on the GraphQL schema?

Comment (tbergq) [reply apps/core/scenes/search/SearchContext.js]: 🤔  Maybe, but probably not within this PR.

File: apps/core/scenes/search/SortTabsWrapper.js (hunk 2 above)
Comment (aalises) [reply apps/core/scenes/search/SortTabsWrapper.js]: Done!

**Comment by tbergq** [comment]
@aalises, Looks like you have a merge conflict

**Comment by aalises** [comment]
@tbergq @JoseDuSV Fixed the conflict!

**Pull request merged**